src/entities/Place/routes/getPlaceMostActiveList.ts

```diff
@@ -60,7 +60,7 @@ export const getPlaceMostActiveList = Router.memo(
       only_highlighted: !!bool(query.only_highlighted),
       positions: query.positions,
       hotScenesPositions: hotScenesPositions,
-      order_by: PlaceListOrderBy.MOST_ACTIVE,
+      order_by: PlaceListOrderBy.UPDATED_AT,
```

Comment:
Are you sure that un the `getPlaceMostActiveList` we should use `UPDATED_AT` as default

Reply:
this change is because doesn't exist column `most_active` in the table, so this will sort in SQL using `updated_at`, then we are applying the sort by `most_active` to the results